generators/app/tasks/installDependencies.js

```diff
@@ -31,9 +32,37 @@ const DEV_DEPENDENCIES = [
   'jest-react-native'
 ];
 
+/**
+ * The eslint config we use may have issues between the different plugins we use.
+ * The solution for this is installing the proper version of each plugin declared in the eslint config package
+ * More info here: https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#usage
+ */
+const getLinterPluginVersions = (projectName, options) =>
+  // Get peer dependencies of eslint-config-airbnb and its versions
+  // This command will return something like the following:
+  // { eslint: '^3.19.0 || ^4.3.0', 'eslint-plugin-jsx-a11y': '^5.1.1', 'eslint-plugin-import': '^2.7.0', 'eslint-plugin-react': '^7.1.0' }
+  runCommand({
+    command: [
+      'npm',
+      ['info', 'eslint-config-airbnb@latest', 'peerDependencies', '--json'],
+      { cwd: `${process.cwd()}/${projectName}` }
+    ],
+    loadingMessage: `Getting eslint plugins' versions`,
+    successMessage: `Successfuly download plugins version info`,
+    failureMessage: `Error getting info of eslint plugins. Turn verbose mode on for detailed logging`,
+    context: options
+  }).then(({ result }) => {
+    const dependencies = JSON.parse(result);
```

Comment:
I don't really agree here. This are only few lines of code that solve a single problem that is in the scope of the function. I'll add some comments to make this clear.